### PR TITLE
feat: Adds support for superscript in help center

### PR DESCRIPTION
**Pull request by nithindavid**
## Description
**Loom** https://www.loom.com/share/fdaf67a0e93c4861a7a4b0560e21f3fd

Fixes # (issue) https://linear.app/chatwoot/issue/CW-1476/support-all-markdown-options-in-the-article-editor

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality not to work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by nithindavid** [comment]
@sojan-official I want change this `CommonMarker.render_html(content).html_safe` to the example mentioned here - https://github.com/gjtorikian/commonmarker#extension-options

But the former method `render_html` isn't showing up here. can you help me?

**Comment by nithindavid** [comment]
@sojan-official I have upgraded the `CommonMarker` gem to `v1.0.0.pre9` and have refactored the code. Let me know if this is good. I'll test all the cases where markdown rendering is involved.

**Comment by scmmishra** [comment]
@nithindavid this change looks good to me, left one small comment  on https://github.com/chatwoot/prosemirror-schema/pull/10

File: app/views/mailers/conversation_reply_mailer/conversation_transcript.html.erb [hunk 1]
```diff
@@ -7,7 +7,11 @@
   <tr>
     <td style="padding-bottom: 16px;">
       <% if message.content %>
-        <%= CommonMarker.render_html(message.content).html_safe %>
+        <%= Commonmarker.to_html(message_content, options:{
```
Comment (sojan-official) [inline app/views/mailers/conversation_reply_mailer/conversation_transcript.html.erb:10]:
```suggestion
        <%= Commonmarker.to_html(message.content, options:{
```

File: app/views/mailers/conversation_reply_mailer/email_reply.html.erb [hunk 2]
```diff
@@ -1,5 +1,9 @@
 <% if @message.content %>
-  <%= CommonMarker.render_html(@message.content).html_safe %>
+  <%= Commonmarker.to_html(message_content, options:{
```
Comment (sojan-official) [inline app/views/mailers/conversation_reply_mailer/email_reply.html.erb:2]:
```suggestion
  <%= Commonmarker.to_html(message.content, options:{
```

File: app/views/mailers/conversation_reply_mailer/reply_with_summary.html.erb [hunk 3]
```diff
@@ -13,7 +13,11 @@
       <% if (message.content_type == 'input_csat' && message.message_type == 'template') %>
         <p>Click <a href="<%= message.conversation.csat_survey_link %>" _target="blank">here</a> to rate the conversation.</p>
       <% elsif message.content.present? %>
-        <%= CommonMarker.render_html(message.content).html_safe %>
+        <%= Commonmarker.to_html(message_content, options:{
```
Comment (sojan-official) [inline app/views/mailers/conversation_reply_mailer/reply_with_summary.html.erb:16]:
```suggestion
        <%= Commonmarker.to_html(message.content, options:{
```

File: app/views/mailers/conversation_reply_mailer/reply_without_summary.html.erb [hunk 4]
```diff
@@ -1,7 +1,11 @@
 <% @messages.each do |message| %>
   <p style="font-family: Roboto,"Helvetica Neue",Tahoma,Arial,sans-serif; text-align: start; unicode-bidi: plaintext;">
     <% if message.content %>
-      <%= CommonMarker.render_html(message.content).html_safe %>
+      <%= Commonmarker.to_html(message_content, options:{
```
Comment (sojan-official) [inline app/views/mailers/conversation_reply_mailer/reply_without_summary.html.erb:4]:
```suggestion
      <%= Commonmarker.to_html(message.content, options:{
```

**Review by sojan-official — COMMENTED** [review]
looks good over all. lets fix the build issues and get this merged
added suggestions.

File: app/views/mailers/conversation_reply_mailer/email_reply.html.erb [hunk 5]
```diff
@@ -1,5 +1,9 @@
 <% if @message.content %>
-  <%= CommonMarker.render_html(@message.content).html_safe %>
+  <%= Commonmarker.to_html(message.content, options:{
```
Comment (sojan-official) [inline app/views/mailers/conversation_reply_mailer/email_reply.html.erb:2]:
```suggestion
  <%= Commonmarker.to_html(@message.content, options:{
```

**Comment by nithindavid** [comment]
@sojan-official Thanks for pointing the mistakes. The build is passing now. Can you take a look?

**Comment by nithindavid** [comment]
Tested the parts where we are using new commonmark gem syntax.

1. Email transcript 
image
image


2. Email notifications
image

3. Reply mailers 
image

**Comment by sojan-official** [comment]
note: the docker builds seems to be failing due to gems missing. @sojan-official  to take a look

**Comment by sojan-official** [comment]
Note: we figured out getting the build to green. But I have removed the force ruby-build which we used to do in docker. This was added for Dialogflow breakage in docker. So we will have to test that before deploying this. 

cc: @nithindavid  @vishnu-narayanan

**Comment by vishnu-narayanan** [comment]
@sojan-official We would need the `force-ruby build` part in Dockerfile to compile `grpc` and `google-protobuf` for the `musl` compatible version needed for our alpine-based docker image.

**Comment by nithindavid** [comment]
Closing in favour of https://github.com/chatwoot/chatwoot/pull/7279

**Pull request closed without merging**